packages/gatsby/src/services/initialize.ts

```diff
@@ -178,6 +178,71 @@ export async function initialize({
     })
   }
 
+  // Setup experiments
+  if (config && config.__experiments) {
+    const experiments = require(`../experiments`).default
+    const configExperiments = config.__experiments
```

Comment:
shouldn't we just go for experiments? __ might be hard for people to discover.

```suggestion
    const configExperiments = config.experiments
```

Reply:
I was going off your original proposal 🤣

But yeah, no reason to add the underscores.